src/integrationTest/java/uk/gov/hmcts/reform/iacasepaymentsapi/testutils/SpringBootIntegrationTest.java

```diff
@@ -23,7 +23,8 @@
     "IA_IDAM_CLIENT_ID=ia",
     "IA_IDAM_SECRET=something",
     "FEES_REGISTER_API_URL=http://127.0.0.1:" + WIREMOCK_PORT + "/fees",
-    "PAYMENT_API_URL=http://127.0.0.1:" + WIREMOCK_PORT + "/payment"
+    "PAYMENT_API_URL=http://127.0.0.1:" + WIREMOCK_PORT + "/payment",
+    "REF_DATA_API_URL=http://127.0.0.1:" + WIREMOCK_PORT + "/refdata/external/v1/organisations/pbas"
```

Comment:
I think you can have here:
`"REF_DATA_API_URL=http://127.0.0.1:" + WIREMOCK_PORT + "/refdata`

stubMethod will look like:
`/refdata/refdata/external/v1/organisations/pbas`

Reply:
Done